public/docs/_examples/pipes/dart/lib/exponential_strength_pipe.dart

```diff
@@ -3,24 +3,18 @@ import 'dart:math' as math;
 import 'package:angular2/angular2.dart';
 
 /*
-* Raise the value exponentially
-* Takes a value that defaults to 0 and an exponent argument that defaults to 1.
-* Checks for value to be a string or number.
-* Usage:
-*   value | exponentialStrength:exponent
-* Example:
-*   {{ 2 |  exponentialStrength:10}}
-*   formats to: 1024
-*/
-
+ * Raise the value exponentially
+ * Takes an exponent argument that defaults to 1.
+ * Usage:
+ *   value | exponentialStrength:exponent
+ * Example:
+ *   {{ 2 |  exponentialStrength:10}}
+ *   formats to: 1024
+ */
 @Pipe(name: 'exponentialStrength')
-@Injectable()
+@Injectable() // FIXME(chalin): unnecessary?
```

Comment:
All core pipes have the `@Injectable()` annotation: https://github.com/angular/angular/blob/master/modules/angular2/src/common/pipes/lowercase_pipe.ts

But I'm not convinced either that it's effectively required. Just try and build your app and see if everything still work without it :)